server/api/rounds.py

```diff
@@ -342,6 +353,68 @@ def sampled_ballot_interpretations_to_cvrs(
     return cvrs
 
 
+def hybrid_contest_strata(
+    contest: Contest, round_one: bool = False
+) -> Tuple[suite.BallotPollingStratum, suite.BallotComparisonStratum]:
+    total_ballots = hybrid_contest_total_ballots(contest)
+    vote_counts = hybrid_contest_choice_vote_counts(contest)
+    assert vote_counts
+    non_cvr_vote_counts = {
+        choice_id: vote_count.non_cvr for choice_id, vote_count in vote_counts.items()
+    }
+    cvr_vote_counts = {
+        choice_id: vote_count.cvr for choice_id, vote_count in vote_counts.items()
+    }
+
+    # For targeted contests, count the number of audited ballot draws so far
+    if contest.is_targeted:
+        num_previous_samples_dict = dict(
+            SampledBallotDraw.query.filter_by(contest_id=contest.id)
+            .join(SampledBallot)
+            .join(Batch)
+            .group_by(Batch.has_cvrs)
+            .values(Batch.has_cvrs, func.count(SampledBallotDraw.ticket_number))
+        )
+    # For opportunistic contests, count the number of audited ballots so far
+    else:
+        num_previous_samples_dict = dict(
+            SampledBallot.query.join(Batch)
+            .join(Jurisdiction)
+            .join(Jurisdiction.contests)
+            .filter_by(id=contest.id)
+            .group_by(Batch.has_cvrs)
+            .values(Batch.has_cvrs, func.count(SampledBallot.id))
+        )
+
+    non_cvr_previous_samples = num_previous_samples_dict.get(False, 0)
```

Comment:
For later, it would be really nice to make True and False here aliased to a more descriptive name like `HAS_CVR` and `NO_CVR` or whatever.

Reply:
interesting, any idea how to do that?

Reply:
we could just define `HAS_CVR = True` and then pass that in.